composer.json

```diff
@@ -20,7 +20,8 @@
         "source": "https://github.com/ramsey/uuid"
     },
     "require": {
-        "php": ">=5.3.3"
+        "php": ">=5.3.3",
+        "paragonie/random_compat": "^2.0"
```

Comment:
Only problem is that it will continue to cause UUID collision problems for folks who are using something that's not updated to 2.0, so I'm a little hesitant to do this, but I'll do it nonetheless, so that I don't cause breaks.